src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/SlimefunItem.java

```diff
@@ -328,6 +314,13 @@ public static SlimefunItem getByItem(ItemStack item) {
 
 	public boolean isItem(ItemStack item) {
 		if (item == null) return false;
+
+		if (item.getItemMeta() != null && this.item.getItemMeta() != null) {
+			String thisId = PersistentDataAPI.getString(this.item.getItemMeta(), Constants.SF_ITEM);
```

Comment:
Instead of getting the id this way you can literally just call "getID()"...

Reply:
Me smart